Repo does not build on 'master' branch

I know recently @bravehorsie made some PRs to change over to JakartaEE dependencies, but it appears that these dependencies not publicly available yet.

> [ERROR] Failed to execute goal on project yasson: Could not resolve dependencies for project org.eclipse:yasson:jar:1.0.3-SNAPSHOT: The following artifacts could not be resolved: jakarta.json.bind:jakarta.json.bind-api:jar:1.0.1, jakarta.json:jakarta.json-api:jar:1.1.5, org.glassfish:jakarta.json:jar:module:1.1.5: Could not find artifact jakarta.json.bind:jakarta.json.bind-api:jar:1.0.1 in central (https://repo.maven.apache.org/maven2) -> [Help 1]

Should these changes be limited to the `EE4J_8` branch so that `master` remains usable?

Also, a general question for @m0mus and @bravehorsie... I have been making PRs against `master` branch for now, but should we be dual-delivering changes to both `master` and `EE4J_8`?

--- a/pom.xml
+++ b/pom.xml
@@ -105,6 +105,14 @@
         <netbeans.hint.jdkPlatform>JDK_9</netbeans.hint.jdkPlatform>
     </properties>
 
+    <!-- TODO: Remove this once JakartaEE 8 artifacts are officially released -->
+    <repositories>
+        <repository>
+            <id>staging</id>
+            <url>https://oss.sonatype.org/content/repositories/staging/</url>
+        </repository>
+    </repositories>
+
     <profiles>
         <profile>
             <id>re-build</id>

